contrib/Dockerfile

```diff
@@ -29,8 +29,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ADD linux /usr/local/bin
 
-EXPOSE 8080
-EXPOSE 9080
```

Comment:
This is the same Dockerfile that is being used for build, test & release. This could break things for folks who choose to simply do `docker run` directly on the image without port-mapping. https://stackoverflow.com/a/22150099 #2 is important.

Reply:
If someone is running a Dgraph Cluster, I am wondering how only exposing alpha ports will make things work. We are not exposing zero ports here.